### PR TITLE
Updating the PR request template

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

Update of the PR request template, prompted by the repeated mention of 'travis'; also updated dev docs on Prettier and ESLint.

# Jira Ticket / Issue #
N/A

# Testing Instructions
* Details for how to test the PR: 
- [x] Tests pass in travis and locally   <--- this will soon change!

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/truss.works/kbasetruss/development
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X
 - [ ] Added the Jira Ticket to the title of the PR e.g. (DATAUP-69 Adds a PR template)
 
 # Testing Instructions
-* Details for how to test the PR: 
-- [ ] Tests pass in travis and locally 
+* Details for how to test the PR:
+- [ ] Tests pass in locally and in GitHub Actions
```
Comment (briehl) [inline .github/pull_request_template.md:13]: typo - Test pass locally

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -23,7 +23,8 @@ e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] Any dependent changes have been merged and published in downstream modules
-- [ ] I have run Black and Flake8 on changed Python Code manually or with git precommit (and the travis build passes)
+- [ ] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook
```
Comment (briehl) [inline .github/pull_request_template.md:26]: 👍  for clarity!

**Review by briehl — CHANGES_REQUESTED** [review]
Looks fine, just a little typo.

File: .github/pull_request_template.md (hunk 1 above)
Comment (ialarmedalien) [reply .github/pull_request_template.md]: Fixed! Thanks for the good 👀

**Review by briehl — APPROVED** [review]
🚀

**Pull request merged**